app/Http/Controllers/config/laraflash.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * This file is part of LaraFlash,
+ * a powerful and flexible flash notifications system.
+ *
+ * @license MIT
+ * @package LaraFlash
+ */
+return [
+    /*
+        |--------------------------------------------------------------------------
+        | Enable Notification Views (COMING SOON)
+        |--------------------------------------------------------------------------
+        |
+        | If true, the package will handle displaying the notifications in the view
+        | otherwise a false value indicates that you only want LaraFlash to manage
+        | the Notification flash object, with you handling the display to users yourself
+        |
+    */
+    'enable_views' => false,
+    /*
+        |--------------------------------------------------------------------------
+        | Default Notification Values
+        |--------------------------------------------------------------------------
+        |
+        | These are the default values that will be set for each notification attribute
+        | if you do not explicitly define
+        | to have a different name, you can do it here.
+        |
+    */
+    'defaults' => [
+        /**
+         * Notification Type
+         *  (ex: danger, success, info, warning, light, dark)
+         */
+        'type' => 'info',
+        /**
+         * Notification Priority
+         *  This is an arbitrary integer to indicate priority over other
+         *  notifications. A higher number is considered higher priority.
+         */
+        'priority' => 5,
+        /**
+         * Notification Title
+         *  (ie. "Success: ", "Warning: ", "Notice: ", null)
+         */
+        'title' => 'Notice: ',
+        /**
+         * Notification Content
+         *  This will probably never revert to this default,
+         *  but is set just in case
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laraflash.php</br>**Line**: 53</br>Closing bracket of array declaration must be on a new line